gherkin/javascript/lib/gherkin/parser.js

```diff
@@ -48,9 +48,6 @@ module.exports = function Parser(builder) {
     if(typeof tokenScanner == 'string') {
       tokenScanner = new TokenScanner(tokenScanner);
     }
-    if(typeof tokenMatcher == 'string') {
```

Comment:
Not sure why this change was made in the generated parser - I assume it was missed out. I assume it is benign.